src/applications/vaos/appointment-list/components/cards/confirmed/VideoVisitProvider.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { getPractitionerDisplay } from '../../../../services/appointment';
+
+export default function VideoVisitProvider({ participants }) {
+  if (!participants) {
+    return null;
+  }
+
+  const practitionerDisplay = getPractitionerDisplay(participants);
+
+  return (
+    <dl className="vads-u-margin--0">
+      <dt className="vads-u-font-weight--bold">You'll be meeting with</dt>
+      <dd>{practitionerDisplay}</dd>
```

Comment:
Nitpick but since it's only used once, you could get rid of `const practitionerDisplay...` above, and do:

`<dd>{getPractitionerDisplay(participants)</dd>`

here.

Reply:
Done